test/js/node/child_process/child_process-node.test.js

```diff
@@ -483,3 +492,261 @@ describe("child_process double pipe", () => {
+      const invalidSecondArgs = [0, true, () => {}, Symbol("t")];
+      invalidSecondArgs.forEach(arg => {
+        expect(() => fork(fixtures.path("child-process-echo-options.js"), arg)).toThrow({
+          code: "ERR_INVALID_ARG_TYPE",
+          name: "TypeError",
+          message: `The \"args\" argument must be of type Array. Received ${arg?.toString()}`,
+        });
+      });
+
+      const argsLists = [undefined, null, []];
+
+      const { mustCall } = createCallCheckCtx(done);
+
+      argsLists.forEach(args => {
+        const cp = fork(fixtures.path("child-process-echo-options.js"), args, {
+          env: { ...process.env, ...expectedEnv },
+        });
+
+        // TODO - bun has no `send` method in the process
+        // cp.on(
+        //   'message',
+        //   common.mustCall(({ env }) => {
+        //     assert.strictEqual(env.foo, expectedEnv.foo);
+        //   })
+        // );
+
+        cp.on(
+          "exit",
+          mustCall(code => {
+            assert.strictEqual(code, 0);
+          }),
+        );
+      });
+    });
+    it("Ensure that the third argument should be type of object if provided", () => {
+      const invalidThirdArgs = [0, true, () => {}, Symbol("t")];
+      invalidThirdArgs.forEach(arg => {
+        expect(() => {
+          fork(fixtures.path("child-process-echo-options.js"), [], arg);
+        }).toThrow({
+          code: "ERR_INVALID_ARG_TYPE",
+          name: "TypeError",
+          message: `The \"options\" argument must be of type object. Received ${arg?.toString()}`,
+        });
+      });
+    });
+  });
+  describe.todo("close", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-close.js
+  });
+  describe.todo("detached", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-detached.js
+  });
+  describe.todo("dgram", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-dgram.js
+  });
+  describe.todo("net", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-net.js
+  });
+  describe.todo("net-server", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-net-server.js
+  });
+  describe.todo("net-socket", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-net-socket.js
+  });
+  describe.todo("no-shell", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-no-shell.js
+  });
+  describe.todo("ref", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-ref.js
+  });
+  describe.todo("stdio", () => {
+    // https://github.com/nodejs/node/blob/v20.5.0/test/parallel/test-child-process-fork-stdio.js
+  });
+  describe.only("fork", () => {
```

Comment:
It should not be using .only, instead .skip

Reply:
Fix it, thank you for finding this